packages/paste-core/components/tooltip/__test__/index.spec.tsx

```diff
@@ -57,6 +57,31 @@ describe('Tooltip', () => {
     });
   });
 
+  describe('Children', () => {
+    it('should not override child provided events such as onBlur or onFocus', async () => {
+      const focusHandlerMock: jest.Mock = jest.fn();
+      const clickHandlerMock: jest.Mock = jest.fn();
+      render(
+        <Theme.Provider theme="console">
+          <Tooltip text="Welcome to Paste!" data-testid="tooltip-children-example">
+            <Button variant="primary" onFocus={focusHandlerMock} onClick={clickHandlerMock}>
+              Open Tooltip
+            </Button>
+          </Tooltip>
+        </Theme.Provider>
+      );
+      await act(async () => {
+        await screen.getByRole('button').focus();
+      });
+      await act(async () => {
+        // @ts-expect-error yes, I know activeElement MIGHT be null, but it's not, OK?
+        await fireEvent.click(document.activeElement);
+      });
+      expect(focusHandlerMock).toHaveBeenCalled();
+      expect(clickHandlerMock).toHaveBeenCalled();
```

Comment:
Can we test that the tooltip still appears too in here even though we override the focus and click event handlers?

Reply:
Mmmmmm yeah, good call. Added them in this test.